### PR TITLE
Medical schema

### PR DESCRIPTION
**Pull request by Peter1907**
Hello, dear code reviewer,

In this project, we added the following tables
- Patients
- Medical_histories
- Treatments
- Invoices
- Invoice_items
- Patient_file (which is a many-to-many relation table)

Provided the appropriate links between tables with foreign keys

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    quantity INT,
+    total_price decimal,
+    invoice_id INT REFERENCES invoices(id) ON DELETE CASCADE,
+    treatment_id INT REFERENCES treatments(id) ON DELETE CASCADE
+)
+
+CREATE TABLE patient_file (
+    history_id INT REFERENCES medical_histories(id),
+    treatment_id INT REFERENCES treatments(id)
+)
```
Comment (MahmoudBakr23) [inline schema_based_on_diagram.sql:44]: - [ ] Great job so far following the ERD to set up your schema and create the tables. However, I think you forgot to indexes for foreign keys. Please do so 🙏 


* `CREATE INDEX ON medical_histories (patient_id);`
* 
* 
etc ...

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello team😃, 

Great work and effort you put into this project so far.

## Highlights 🚀

There are some excellent points that you have made perfectly. Keep up! 💪
- Well done following the ERD to create your tables so far.
- Descriptive PR and README file

**Optional**
- Please fix the MIT link for a professional README file

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, not criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (Reem-lab) [reply schema_based_on_diagram.sql:44]: - [x] Great job adding these indexes but you still need to add the other indexes for the another tables as well so kindly according to that add the rest of indexes on `medical_hitsories` and `invoices`  and `invoices_items` tables
invoices item should be once for `invoice_id` and once for `treatment_id`. 💪🏻🚀

**Review by Reem-lab — CHANGES_REQUESTED** [review]
# Hi Team, 👋🏻👋🏻

Great work on making the changes requested by a previous reviewer 👏🔥💯💯
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
📌 Followed Gitflow ✅
📌 Great working on the functionality ✅
📌Descriptive Pull request ✅


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MarcoAurelioAntonio** [comment]
Hi @Reem-lab !, nice to write to you!... I get confused... maybe you can help us.
I thought we already create all the others FK's **inside of every table**. If is wrong or is bad practice our code, please tell us how to do it well 🙏 

@MahmoudBakr23 Hi!... we apreciate if you can help us too.

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle: :1st_place_medal: :100:
success

## Hey @Peter1907, @MarcoAurelioAntonio :wave:

Great work on implementing the requested changes. Your project is complete. :blush:
There is nothing else to say about your project. :+1:


## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following GitFlow
✔️ Following requirements

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**